NitroxServer/Player.cs

```diff
@@ -32,6 +32,15 @@ public Player(PlayerContext playerContext, NitroxConnection connection, Vector3
             GameObjectId = new NitroxId();
         }
 
+        public Player(PlayerContext playerContext, NitroxConnection connection, Vector3 position, NitroxId playerId, Optional<NitroxId> subRootId)
```

Comment:
Can you move this constructor above the other one,  and let the other one use this one? That would look like:

```C#
public Player(PlayerContext playerContext, NitroxConnection connection, Vector3 position, Optional<NitroxId> subRootId)
        : this(playerContext, connection, position, new NitroxId(), subRootId)
        {
        }
```

Reply:
I did not think about that in that moment